src/main/java/core/basesyntax/service/parsefileinfo/FruitInfo.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax.service.parsefileinfo;
+
+import core.basesyntax.service.functionalityexpansion.ActivityType;
+
+public record FruitInfo(ActivityType activityType, String name, int quantity) {
```

Comment:
just to be a bit more clear, because fruit info would be some fields like color weight, size etc., but we have here activity and amount, which changes the main thing that this class represents to some transaction info
```suggestion
public record FruitTransactionInfo(ActivityType activityType, String name, int quantity) {
// or just FruitTransaction
```